### PR TITLE
Ember.get and .set work with global paths in 1.0 mode

### PR DESCRIPTION
**Pull request by jamesarosen**
@shajith @jish 

When `ACCESSORS` is set to `"0.9"` or `"0.9-dotted-properties"`,
- `Em.getPath('Foo.bar')` returns `window.Foo.bar`
- `Em.setPath('Foo.bar', 'baz')` sets `window.Foo.bar` to `"baz"`

When `ACCESSORS` is set to `"1.0-no-warn"` or `"1.0"`, the `getPath` and `setPath` versions continued to work, but `get` and `set` didn't work with global paths. Now they do, just like in Ember 1.0.

This PR also includes two small drive-by fixes:
- `scripts/browser_tests` had a typo
- `backported/view_state_tests` was in the wrong package

### CODE REVIEW

File: packages/ember-metal/lib/accessors.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ get = function get(obj, keyName) {
     Ember.deprecate("The behavior of `get` has changed in Ember 1.0. It will no longer support keys with periods in them.", !hasDot);
   }
 
+  if (!obj && is10 && Ember.isGlobalPath(keyName)) {
+    obj = window;
+  }
```
Comment (jish) [inline packages/ember-metal/lib/accessors.js]: I might be following this incorrectly, but how could we get this far down and still have `!obj` evaluate truthy?

``` js
Em.get(foo, 'Global.bar'); // obj is foo
Em.get('Global.bar');      // obj is Ember
```

Comment (jamesarosen) [inline packages/ember-metal/lib/accessors.js]: Right. I did this correctly below (in the `USE_ACCESSORS` version of `get`), but not hear. This was an earlier attempt that I forgot to fix.

**Comment by jamesarosen** [comment]
@jish :haircut:

**Comment by jish** [comment]
+1

**Comment by jamesarosen** [comment]
Merged manually

**Pull request closed without merging**